src/app/components/modal-dialog/modal-dialog.component.spec.ts

```diff
@@ -16,10 +16,20 @@ describe('ModalDialogComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(ModalDialogComponent);
     component = fixture.componentInstance;
+    component.message = "testMessage";
+    component.title = "testTitle";
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('values test', () => {
+    const nativeElement = fixture.nativeElement;
+    const title = nativeElement.querySelector('h1');
+    const message = nativeElement.querySelector('p');
+    expect(title.textContent).toBe("testTitle");
+    expect(message.textContent).toBe("testMessage");
+  })
 });
```

Comment:
add more tests please, regarding component interactions, like you click something or use some public API, then something happens which component should handle, and your test will validate how it handled the interaction so it'll be correct